nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -181,19 +328,57 @@ def test_instantiation_error_plotly_surface_figure(input_obj):
         PlotlySurfaceFigure(input_obj)
 
 
-def test_set_view_plot_surf_errors():
-    from nilearn.plotting.surf_plotting import (_set_view_plot_surf_matplotlib,
-                                                _set_view_plot_surf_plotly)
+
+@pytest.mark.parametrize(
+    "view,is_valid",
+    [
+        ("lateral", True),
+        ("medial", True),
+        ("latreal", False),
+        ((100, 100), True),
+        ((100, 100, 1), False),
+    ]
+)
+def test_check_view_is_valid(view, is_valid):
+    from nilearn.plotting.surf_plotting import _check_view_is_valid
+    assert _check_view_is_valid(view) is is_valid
+
+
+@pytest.mark.parametrize(
+    "hemi,is_valid",
+    [
+        ("left", True),
+        ("right", True),
+        ("lft", False),
+    ]
+)
+def test_check_hemisphere_is_valid(hemi, is_valid):
+    from nilearn.plotting.surf_plotting import _check_hemisphere_is_valid
+    assert _check_hemisphere_is_valid(hemi) is is_valid
+
+
+@pytest.mark.parametrize("hemi,view", [("foo", "medial"), ("bar", "anterior")])
+def test_get_view_plot_surf_hemisphere_errors(hemi, view):
+    from nilearn.plotting.surf_plotting import (_get_view_plot_surf_matplotlib,
+                                                _get_view_plot_surf_plotly)
     with pytest.raises(ValueError,
-                       match="hemi must be one of"):
-        _set_view_plot_surf_matplotlib("foo", "medial")
-        _set_view_plot_surf_plotly("bar", "anterior")
+                       match="Invalid hemispheres definition"):
+        _get_view_plot_surf_matplotlib(hemi, view)
     with pytest.raises(ValueError,
-                       match="view must be one of"):
-        _set_view_plot_surf_matplotlib("left", "foo")
-        _set_view_plot_surf_matplotlib("right", "bar")
-        _set_view_plot_surf_plotly("left", "foo")
-        _set_view_plot_surf_plotly("right", "bar")
+                       match="Invalid hemispheres definition"):
+        _get_view_plot_surf_plotly(hemi, view)
+
+
+@pytest.mark.parametrize("hemi,view", [("left", "foo"), ("right", "bar")])
+def test_get_view_plot_surf_view_errors(hemi, view):
+    from nilearn.plotting.surf_plotting import (_get_view_plot_surf_matplotlib,
+                                                _get_view_plot_surf_plotly)
+    with pytest.raises(ValueError,
+                       match="Invalid view definition"):
+        _get_view_plot_surf_matplotlib(hemi, view)
+    with pytest.raises(ValueError,
+                       match="Invalid view definition"):
+        _get_view_plot_surf_plotly(hemi, view)
```

Comment:
Seems like you can parametrize on the function also (you'll need to import them at the top then...).